python/cudf/cudf/core/series.py

```diff
@@ -142,6 +142,9 @@ def _column(self, value):
     def __contains__(self, item):
         return item in self._index
 
+    def __reduce__(self):
+        return (Series, (self._data, self.index))
```

Comment:
Can this be pushed down to the Table / Frame class?

Reply:
See comment in `dataframe.py`